package.json

```diff
@@ -30,14 +30,15 @@
     "combined-stream": "^1.0.5",
     "csv-parse": "^1.0.0",
     "fs-extra": "^2.0.0",
+    "download-file-sync": "^1.0.4",
     "iso3166-1": "^0.2.5",
     "joi": "^10.0.6",
     "lodash": "^4.5.1",
     "node-version-checker": "^2.0.0",
     "pelias-config": "2.7.1",
     "pelias-dbclient": "2.0.0",
     "pelias-logger": "0.1.0",
-    "pelias-model": "4.4.0",
+    "pelias-model": "git://github.com/pelias/model.git#add-postalcode",
```

Comment:
don't forget to change this back. Ideally it would be rebased out, otherwise on the off chance we want to look at the history of the dependencies, a much more thorough investigation will be required to know when this version was last changed from an actual version, rather than flipped around for testing.